app/models/lookup_keys/lookup_key.rb

```diff
@@ -103,7 +103,8 @@ def supports_uniq?
   end
 
   def to_param
-    Parameterizable.parameterize("#{id}-#{key}")
+    search_key = key.tr(' ','_') unless key.nil?
```

Comment:
If we don't allow whitespace using validation, there won't ever be a key containing spaces once the migration runs.